doc/cephfs/mount-using-fuse.rst

```diff
@@ -9,17 +9,10 @@ clients can be more manageable, especially while upgrading CephFS.
 Prerequisites
 =============
 
-Complete General Prerequisites
-------------------------------
 Go through the prerequisites required by both, kernel as well as FUSE mounts,
 in `Mount CephFS: Prerequisites`_ page.
 
-``fuse.conf`` option
---------------------
-
-#. If you are mounting Ceph with FUSE not as superuser/root user/system admin
-   you would need to add the option ``user_allow_other`` to ``/etc/fuse.conf``
-   (under no section in the conf).
+.. note:: Mounting CephFS using FUSE requires superuser privileges.
```

Comment:
nit: To emphasize that this change is relatively recent.
```suggestion
.. note:: Mounting CephFS using FUSE requires superuser privileges now.
```

Reply:
Go further and explain why it requires superuser privileges.

Reply:
Done